File: Problems/Algorithms/26. Remove Duplicates from Sorted Array/RemoveDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int ans = 0;
        
        for (int i = 0; i < nums.length; i++) {
            if (i == 0 || nums[i] != nums[i - 1]) {
                nums[ans++] = nums[i];
            }
        }

        return ans;
    }
}

File: Problems/Algorithms/26. Remove Duplicates from Sorted Array/remove_duplicates.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ans = 0
        for i in range(len(nums)):
            if nums[i] != nums[ans]:
                ans += 1
                nums[ans] = nums[i]
        return ans + 1
